test/test_frame_2d.py

```diff
@@ -37,6 +37,15 @@ def test_info(physical_camera_frame_2d):
     assert isinstance(info, zivid.FrameInfo)
 
 
+@pytest.mark.physical_camera
+def test_camera_info(physical_camera_frame_2d):
+    from zivid.camera_info import CameraInfo
+
+    camera_info = physical_camera_frame_2d.camera_info
+    assert camera_info
+    assert isinstance(camera_info, CameraInfo)
```

Comment:
Looks OK, I would have liked some more testing of the properties of the CameraInfo, such as the model and serialnumber. I see also test_frame.py has the same test though, so then both could be improved a bit.

Reply:
The actual properties are tested in `test/test_camera_info.py`.